modules/apps/data-engine/data-engine-test/src/testFunctional/tests/macros/DataEngineUpgrade.macro

```diff
@@ -46,7 +46,7 @@ definition {
 			webContentDate = "02/22/2021",
 			webContentDateFieldLabel = "Date",
 			webContentNumeric = "1.1",
-			webContentNumericFieldLabel = "Decimal",
+			webContentNumericFieldLabel = "Number",
```

Comment:
Hey @Odrakir1 , next time you can amend these fixes into the commit where you wrote the code. Thanks!